diff --git a/dist/client/components/CommitRegularLine.vue b/dist/client/components/CommitRegularLine.vue
index 4008495b51c2ce7fe1300cbc7c5ce3d6d5463227..b9ef3b59be35c5377c5733ae1b4a7947332be961 100644
--- a/dist/client/components/CommitRegularLine.vue
+++ b/dist/client/components/CommitRegularLine.vue
@@ -55,7 +55,7 @@ function formatCommittedOn(timestamp: number): string {
     <span>-</span>
     <span>
       <span class="text-sm <sm:text-xs" v-html="renderCommitMessage(commit.repo_url || 'https://github.com/example/example', commit.message)" />
-      <span v-if="options.displayAuthorsInsideCommitLine" class="my-1 ml-3 gap-1">
+      <span v-if="options.displayAuthorsInsideCommitLine" class="vp-nolebase-git-changelog-authors my-1 ml-3">
         <template
           v-for="a of commit.authors"
           :key="a.name"
@@ -63,11 +63,15 @@ function formatCommittedOn(timestamp: number): string {
           <a
             v-if="(typeof a.url !== 'undefined')"
             :href="a.url"
-            class="no-icon"
+            class="vp-nolebase-git-changelog-author no-icon"
           >
             <img :src="a.avatarUrl" :alt="`The avatar of contributor named as ${a.name}`" class="vp-nolebase-git-changelog-commit-avatar inline-block h-6 w-6 rounded-full v-middle">
+            <span class="vp-nolebase-git-changelog-author-name text-sm <sm:text-xs v-middle">{{ a.name }}</span>
           </a>
-          <img v-else :src="a.avatarUrl" :alt="`The avatar of contributor named as ${a.name}`" class="vp-nolebase-git-changelog-commit-avatar inline-block h-6 w-6 rounded-full v-middle">
+          <span v-else class="vp-nolebase-git-changelog-author">
+            <img :src="a.avatarUrl" :alt="`The avatar of contributor named as ${a.name}`" class="vp-nolebase-git-changelog-commit-avatar inline-block h-6 w-6 rounded-full v-middle">
+            <span class="vp-nolebase-git-changelog-author-name text-sm <sm:text-xs v-middle">{{ a.name }}</span>
+          </span>
         </template>
       </span>
       <ClientOnly>
@@ -82,6 +86,14 @@ function formatCommittedOn(timestamp: number): string {
 <style scoped>
 .vp-nolebase-git-changelog-commit-avatar {
   border: 2px solid var(--vp-c-bg-soft);
-  margin-left: -0.5em;
+}
+
+.vp-nolebase-git-changelog-author {
+  margin-right: 0.5em;
+  white-space: nowrap;
+}
+
+.vp-nolebase-git-changelog-author-name {
+  margin-left: 0.25em;
 }
 </style>
